docs/hardware/1-home.md

```diff
@@ -58,4 +83,4 @@ low development effort.
 
 If you're interested in using a board from this category, you can reach out to us on
 [Discord](https://golioth.io/discord), and we can help you with next steps to
```

Comment:
Do we want to change this to the forum instead of discord? (I recall a general desire to shift folks there)

Reply:
Out of the scope of this PR but I can add it.